cluster/gce/windows/k8s-node-setup.psm1

```diff
@@ -1584,13 +1597,16 @@ function Configure_Containerd {
     replace('CNI_CONF_DIR', ${env:CNI_CONFIG_DIR})
 }
 
-# Register and start containerd service.
+# Register if needed and start containerd service.
 function Start_Containerd {
-  Log-Output "Creating containerd service"
-  & containerd.exe --register-service --log-file ${env:LOGS_DIR}/containerd.log
+  # Do the registration only if the containerd service does not exist.
+  if (Get-WMIObject -Class Win32_Service -Filter  "Name='containerd'" -eq $null) {
+    Log-Output "Creating containerd service"
+    & containerd.exe --register-service --log-file ${env:LOGS_DIR}/containerd.log
+  }
+
   Log-Output "Starting containerd service"
-  Start-Service containerd
-}
+  Restart-Service containerd
```

Comment:
Why was this changed to Restart-Service?

Reply:
Because it was assuming that the runtime is always installed fresh through the script, now the service could already be existing and running, so we'll need to have a restart after updating its config.